pkg/testframework/config.go

```diff
@@ -39,3 +44,40 @@ func GetConfig() (*rest.Config, error) {
 
 	return nil, fmt.Errorf("could not locate a kubeconfig")
 }
+
+// Reset
+func Reset(t *testing.T, client *Clientset, logLevel v1.LogLevel) {
+	//todo: need logic
+}
+
+// Set Brand on the config
+func SetBrand(t *testing.T, client *Clientset, brand v1.Brand) error {
+	operatorConfig, err := client.Consoles().Get(consoleapi.ConfigResourceName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	// set it
+	operatorConfig.Spec.Customization.Brand = brand
+	_, err = client.Consoles().Update(operatorConfig)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func GetBrand(t *testing.T, client *Clientset) string {
```

Comment:
Should the `GetConsoleConfigmap()` happen outside this func?  I would expect `GetBrand(configmap)` to be a helper to return a `Brand`, not sure if it should encapsulate an `http` request as well.

Reply:
Well, part of the purpose of the function is to also reduce repeated code. So if I did not do the `GetConsoleConfigMap` inside this function, I would have to call that function before each call to `GetBrand` in each of my tests.  From an OOP standpoint, the getter should be simpler, but here this is a utility function that is getting called from multiple places.